### PR TITLE
Add dependency on and compatibility with PyJWT 2.0

### LINKED ISSUE #118: PyJWT 2.0 not compatible
The current version is not compatible with version 2.0 of PyJWT, since jwt.ExpiredSignature has been renamed to jwt.ExpiredSignatureError (used in line 17 in backend.py). 

Suggested suggestions:
- Either realign with PyJWT 2.0
- Cahnge install_requires in setup.py from **'pyjwt >= 1.0.1',** to **'pyjwt >= 1.0.1, < 2.0',**

### PR DESCRIPTION
**Pull request by SanderTuit**
Fixes #118

### CODE REVIEW

**Comment by SanderTuit** [comment]
As far as I remember, this is the first pull request I created on a public project. So if there's anything wrong, please let me know. The Travis tests all seem to fail, because PyJWT 2.0.0 is not found. However 2.0.0 [was released](https://pypi.org/project/PyJWT/#history) on december 22. I'm not sure if there's a cache that should be emptied or a package index that can be refreshed.

**Comment by JonasKs** [comment]
Hi! That’s awesome, happy you’re on the “dark side” with us 😉 Looks good. I’ll just verify locally before I merge, but I’ll do that today. 

As for people waiting for this PR to be merged @ingvaldlorentzen has provided a work around in #118

**Comment by JonasKs** [comment]
Sorry, busy day. I’ll hopefully have time tomorrow, and if not I’ll 100% have time this weekend.

File: setup.py [hunk 1]
```diff
@@ -44,9 +44,9 @@
         'Development Status :: 5 - Production/Stable',
     ],
     install_requires=[
-        'pyjwt >= 1.0.1',
+        'pyjwt>=2.0',
```
Comment (JonasKs) [inline setup.py:47]: I've done some research, and the bump here is actually not needed. The `ExpiredSignatureError` exists in `1.0.1` as you can see [here](https://github.com/jpadilla/pyjwt/blob/1.0.1/jwt/exceptions.py) and [here](https://github.com/jpadilla/pyjwt/blob/1.0.1/jwt/__init__.py).


So, in other words, this change can be reverted, and the only change needed is the change from `ExpiredSignature` to `ExpiredSignatureError` as you've done in `backend.py`.
When this has been done, I'll merge this PR. :+1:

**Comment by JonasKs** [comment]
Perfect, thank you @SanderTuit ! 

I'll have to do a few things before I can publish this to PyPI, but it'll happen on Saturday.

**Comment by JonasKs** [comment]
`1.4.0` has been released to PyPI.

**Pull request merged**